src/Resources/config/services.php

```diff
@@ -57,7 +57,7 @@
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\HttpFoundation\RequestStack;
-use Symfony\Component\Security\Core\Security;
+use Symfony\Bundle\SecurityBundle\Security;
```

Comment:
This is not a proper solution. The bundle currently supports >= Symfony 5.3, but this new class was only introduced in Symfony 6.2 so this PR would actually break the bundle for all apps that are not on Symfony 6.2